apiserver/Makefile

```diff
@@ -147,7 +147,7 @@ ifneq (${OPERATOR_IMAGE_TAG}, latest)
 	$(ENGINE) pull kuberay/security-proxy:$(OPERATOR_IMAGE_TAG)
```

Comment:
I think this would more usable if the use of the "OPERATOR_IMAGE_TAG" would not be overloaded (used to to control the experimental security image). I would strongly recomend that we create a separate variable (`SECURITY_IMAGE_TAG`) with a default value of `latest`. This way one would have the flexibility of mixing and matching all the artifacts deployed. If this done the documentation would need to be updated.

Reply:
Done